ramls/service-points-user.raml

```diff
@@ -10,7 +10,7 @@ documentation:
 
 types:
   servicePointsUser: !include servicepointsuser.json
-  servicepointsusers: !include servicepointsusers.json
+  servicePointsUsers: !include servicepointsusers.json
```

Comment:
Why has the case of this name been changed?

Reply:
In order to have the class generated with correct camel style.

Reply:
Ah, I didn't know that.